app/domain/people/neuanmeldungen/approve.rb

```diff
@@ -5,65 +5,70 @@
+  #
+  class Approve < Base
+    def call
+      applicable_roles.each do |role|
+        Role.transaction do
+          role.destroy!
+          create_approved_role(role)
         end
       end
+    end
 
-      private
+    private
 
-      def approved_roles_group
-        @approved_roles_group ||=
-          group
-            .parent.children.without_deleted
-            .find_by(type: APPROVED_NEUANMELDUNGEN_GROUP.sti_name) ||
-          APPROVED_NEUANMELDUNGEN_GROUP.create!(parent: group.parent)
-      end
+    def approved_roles_group
+      @approved_roles_group ||=
+        group
+          .parent.children.without_deleted
+          .find_by(type: APPROVED_NEUANMELDUNGEN_GROUP.sti_name) ||
+        APPROVED_NEUANMELDUNGEN_GROUP.create!(parent: group.parent)
+    end
 
-      def create_approved_role(role)
-        return if approved_roles_group.roles
-          .where(type: APPROVED_NEUANMELDUNGEN_ROLE.sti_name, person_id: role.person_id)
-          .exists?
+    def create_approved_role(role)
+      return if approved_roles_group.roles
+        .where(type: APPROVED_NEUANMELDUNGEN_ROLE.sti_name, person_id: role.person_id)
+        .exists?
 
-        APPROVED_NEUANMELDUNGEN_ROLE.create!(
-          group: approved_roles_group,
-          person: role.person,
-          beitragskategorie: role.beitragskategorie,
-          created_at: role.created_at,
-          delete_on: role.delete_on
-        )
-        generate_invoice(role)
-      end
+      APPROVED_NEUANMELDUNGEN_ROLE.create!(
+        group: approved_roles_group,
+        person: role.person,
+        beitragskategorie: role.beitragskategorie,
+        created_at: role.created_at,
+        delete_on: role.delete_on
+      )
+      generate_invoice(role)
+      send_confirmation_mail(role.person)
+    end
 
-      def generate_invoice(role)
-        return if role.beitragskategorie&.family? && !role.person.sac_family_main_person
+    def generate_invoice(role)
+      return if role.beitragskategorie&.family? && !role.person.sac_family_main_person
 
-        invoice = ExternalInvoice::SacMembership.create!(
-          person: role.person,
-          state: :draft,
-          year: Date.current.year,
-          issued_at: Date.current,
-          sent_at: Date.current,
-          link: role.layer_group
-        )
-        Invoices::Abacus::CreateInvoiceJob.new(invoice, Date.current, new_entry: true).enqueue!
-      end
+      invoice = ExternalInvoice::SacMembership.create!(
+        person: role.person,
+        state: :draft,
+        year: Date.current.year,
+        issued_at: Date.current,
+        sent_at: Date.current,
+        link: role.layer_group
+      )
+      Invoices::Abacus::CreateInvoiceJob.new(invoice, Date.current, new_entry: true).enqueue!
+    end
+
+    def send_confirmation_mail(person)
+      return unless person.sac_family_main_person?
+
+      People::NeuanmeldungenMailer.approve(person, @group).deliver_later
```

Comment:
Email sollte auch wieder in der sprache die auf der Person gesetzt ist versendet werden

Reply:
das wird sie hier https://github.com/hitobito/hitobito_sac_cas/pull/938/files#diff-dd40239cf9d0f7c4f855636b1d8b597578a512d2fff5465d2e66ffec6b4843acR25